src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -120,62 +157,428 @@ def _load_iges(self):
+          N Arrays:   1
+
+        Return a list of keypoints as indiviudal grids
+
+        >>> keypoints = mapdl.geometry.get_keypoints(return_as_list=True)
+        >>> keypoints
+        [PolyData (0x147ca4be...rrays:   1, PolyData (0x147d5b8e...rrays:   1, PolyData (0x1491a42e...rrays:   1, PolyData (0x1491a440...rrays:   1, PolyData (0x1491a47c...rrays:   1, PolyData (0x1491a470...rrays:   1, PolyData (0x1491a4b2...rrays:   1, PolyData (0x1491a4e2...rrays:   1, PolyData (0x1491a49a...rrays:   1, PolyData (0x1491a4f4...rrays:   1, PolyData (0x1491a458...rrays:   1, PolyData (0x1491a4b8...rrays:   1, PolyData (0x1491a4dc...rrays:   1, PolyData (0x1491a506...rrays:   1, ...]
+
+        Return the keypoints coordinates as a numpy array:
+
+        >>> keypoints = mapdl.geometry.get_keypoints(return_as_array=True)
+        array([[ 0.00000000e+00,  1.77800000e-02, -3.18000000e-03],
+               [ 0.00000000e+00, -7.62000000e-03, -3.18000000e-03],
+               [ 1.58750000e-02,  1.77800000e-02, -3.18000000e-03],
+               [ 1.58750000e-02, -7.62000000e-03, -3.18000000e-03],
+               [ 0.00000000e+00, -7.62000000e-03,  0.00000000e+00],
+        ...
+
+        When returning an array, you can also choose to output the keypoint
+        ids as the first column:
+
+        >>> keypoints = mapdl.geometry.get_keypoints(return_ids_in_array=True)
+        array([[ 1.00000000e+00,  0.00000000e+00,  1.77800000e-02,
+                -3.18000000e-03],
+               [ 2.00000000e+00,  0.00000000e+00, -7.62000000e-03,
+                -3.18000000e-03],
+               [ 3.00000000e+00,  1.58750000e-02,  1.77800000e-02,
+                -3.18000000e-03],
+               [ 4.00000000e+00,  1.58750000e-02, -7.62000000e-03,
+                -3.18000000e-03],
+               [ 5.00000000e+00,  0.00000000e+00, -7.62000000e-03,
+                 0.00000000e+00],
+               [ 6.00000000e+00,  1.58750000e-02, -7.62000000e-03,
+        ...
+
+        """
+        if return_as_array or return_ids_in_array:
+            if return_ids_in_array:
+                keyp = np.array(self._keypoints[0])
+                keyp_num = np.array(self._keypoints[1])
+                return np.hstack((keyp_num.reshape((-1, 1)), keyp))
+            else:
+                return np.array(self._keypoints[0])
+
+        keypoints, kp_num = self._keypoints
+
+        if return_as_list:
+            keypoints_ = []
+
+            for each_point, each_id in zip(keypoints, kp_num):
+                keypoints_ps = pv.PolyData([each_point])
+                keypoints_ps["entity_num"] = np.array(each_id).reshape((1))
+                keypoints_.append(keypoints_ps)
+
+            return keypoints_
+
+        else:
+            keypoints_pd = pv.PolyData(keypoints)
+            keypoints_pd["entity_num"] = kp_num
+            return keypoints_pd
 
     @property
-    def _lines(self):
+    def _lines(self) -> List[pv.PolyData]:
         """Returns lines cache"""
```

Comment:
```suggestion
        """Cache of the lines."""
```

Reply:
I didn't know about not allowing verbs in the property descriptions. :)